include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -542,6 +559,9 @@ ORT_DEFINE_RELEASE(ValueInfo);
 ORT_DEFINE_RELEASE(Node);
 ORT_DEFINE_RELEASE(Graph);
 ORT_DEFINE_RELEASE(Model);
+#if !defined(ORT_MINIMAL_BUILD)
```

Comment:
>ORT_MINIMAL_BUILD [](http://example.com/codeflow?start=13&length=17)

My understanding ORT_MINIMAL_BUILD is a build option for the ORT, but not for the customer source code.
They code will probably not have a notion of MINIMAL build. So this will always be undefined.
I suggest we do not leak our internal defines to public headers.